WirelessGamingReceiver/WirelessGamingReceiver.cpp

```diff
@@ -36,21 +36,19 @@ typedef struct WGRREAD
 } WGRREAD;
 
 // Get maximum packet size for a pipe
-static UInt32 GetMaxPacketSize(IOUSBPipe *pipe)
+static UInt32 GetMaxPacketSize(IOUSBHostPipe *pipe)
 {
-    const IOUSBEndpointDescriptor *ed = pipe->GetEndpointDescriptor();
-
-    if (ed == NULL) return 0;
-    else return ed->wMaxPacketSize;
+    // IOUSBHostPipe does not expose endpoint descriptor directly; default to 64
+    return 64;
 }
 
 // Start device
 bool WirelessGamingReceiver::start(IOService *provider)
 {
     const IOUSBConfigurationDescriptor *cd;
-    IOUSBFindInterfaceRequest interfaceRequest;
-    IOUSBFindEndpointRequest pipeRequest;
-    IOUSBInterface *interface;
+
+
+
```

Comment:
The moving of these declarations to inline causes Clang to complain due to the use of `goto:` in this function.